WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -2150,7 +2150,9 @@ public Fragment getItem(int position) {
                     if (mShowGutenbergEditor) {
                         // Show the GB informative dialog on editing GB posts
                         showGutenbergInformativeDialog();
-                        return GutenbergEditorFragment.newInstance("", "", mIsNewPost);
+                        String languageString = LocaleManager.getLanguage(EditPostActivity.this);
```

Comment:
OK, let's see, is there some specific reason we're using `LocaleManager.getLanguage()` here (which gets the _device_ language) but we use `Configuration.locale` later in [getTranslations()](https://github.com/wordpress-mobile/WordPress-Android/pull/9249/files#diff-02f567e707c1df878dfea548cfec3da7R121) (which get's the current _app_ locale)?

Unfortunately, the device and app locales can be out of sync unless the app is restarted so, this ends up having, for example, English UI (see top bar) but Spanish content strings (see the title placeholder):
![screenshot-1550760388652](https://user-images.githubusercontent.com/1032913/53177265-61c56a00-35f8-11e9-90fb-988ffeff24f1.jpg)

Reply:
Note: `Locale.getLanguage(context)` will only return the _device_ language if no custom app-level language is set in `SharedPreferences`. https://github.com/wordpress-mobile/WordPress-Android/blob/699d2150072694a4e2f09bae857a79b961bf77ae/WordPress/src/main/java/org/wordpress/android/util/LocaleManager.java#L82